diff -dur celery-3.1.19.orig/docs/conf.py celery-3.1.19/docs/conf.py
--- celery-3.1.19.orig/docs/conf.py	2015-10-26 21:04:39.000000000 +0100
+++ celery-3.1.19/docs/conf.py	2015-11-17 09:53:06.000000000 +0100
@@ -76,11 +76,11 @@
 add_function_parentheses = True
 
 intersphinx_mapping = {
-    'python': ('http://docs.python.org/dev', None),
-    'kombu': ('http://kombu.readthedocs.org/en/latest/', None),
-    'djcelery': ('http://django-celery.readthedocs.org/en/latest', None),
-    'cyme': ('http://cyme.readthedocs.org/en/latest', None),
-    'amqp': ('http://amqp.readthedocs.org/en/latest', None),
+    'python': ('http://docs.python.org/dev', "python-objects.inv"),
+    'kombu': ('http://kombu.readthedocs.org/en/latest/', "kombu-objects.inv"),
+    'djcelery': ('http://django-celery.readthedocs.org/en/latest', "djcelery-objects.inv"),
+    'cyme': ('http://cyme.readthedocs.org/en/latest', "cyme-objects.inv"),
+    'amqp': ('http://amqp.readthedocs.org/en/latest', "amqp-objects.inv"),
 }
 
 # The name of the Pygments (syntax highlighting) style to use.
